pkg/storage/cacher.go

```diff
@@ -405,10 +408,15 @@ func (c *cacheWatcher) add(event watchCacheEvent) {
 }
 
 func (c *cacheWatcher) sendWatchCacheEvent(event watchCacheEvent) {
-	curObjPasses := event.Type != watch.Deleted && c.filter(event.Object)
+	curObjPasses := false
```

Comment:
Sorry - I don't understand.
To clarify from my side: the intention was that there will be no change. And the check "event.Type != watch.Deleted" was moved to line 413 (it didn't disappear).